### DISCUSSION TITLE
部署问题

### DISCUSSION THREAD

**Discussion by deftlakers** [wzdavid/ThinkRAG#6] [Q&A]
作者你好，最近在学习这个优秀的项目。但是配置环境后，启动app，网页打不开，初始界面也进不去，请问下是什么原因。

**Comment by wzdavid** [comment]
可否提供详细的信息，控制台输出是什么？如果没有提前从HuggingFace下载模型，系统启动时需要很长时间下载的

**Comment by deftlakers** [comment]
感谢作者的回复，目前问题已经解决，再次感谢您的开源贡献。



---原始邮件---
发件人: "David ***@***.***&gt;
发送时间: 2024年11月24日(周日) 上午9:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [wzdavid/ThinkRAG] 部署问题 (Discussion #6)




 
可否提供详细的信息，控制台输出是什么？如果没有提前从HuggingFace下载模型，系统启动时需要很长时间下载的
